frame/contracts/src/tests.rs

```diff
@@ -415,6 +415,7 @@ fn instantiate_and_call_and_deposit_event() {
 	let (wasm, code_hash) = compile_module::<Test>(CODE_RETURN_FROM_START_FN).unwrap();
 
 	ExtBuilder::default().existential_deposit(100).build().execute_with(|| {
+		System::set_block_number(1);
```

Comment:
should just do all those calls in `ExtBuilder`, instead of individually call it in every tests, most of the tests should be run in block 1, not zero anyway.

Reply:
Yup good idea